src/script/vco-ie/main_warhammer/wh2_main_skv_clan_moulder/missions.lua

```diff
@@ -1,150 +1,150 @@
-local missions = {

```

Comment:
@OhManTFE 
Did you change something regarding Clan Moulder? And Great Orthodoxy? 

I am not sure what is happening here. What do you think @Mjb141? Maybe the end of line?

Reply:
Not in this packfile.

I believe I made changes in the myedits packfile. If I was changing the lua file it would have been to reorder things for the red text overhaul.